crates/hyved/systemd/debian/hyved@.hyveos-dhcp-client.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=HyveOS DHCP Client (%i)
+Requires=hyved@%i.hyveos-batman.service
+After=hyved@%i.hyveos-batman.service
```

```suggestion
Requires=hyveos-batman@%i.service
After=hyveos-batman@%i.service
```

`cargo-deb` uses very weird naming for systemd services with parameters for some reason (I had to look into the source code of `cargo-deb` to find out how to name the services so that they would appear at the correct place inn the `.deb` file...).